src/components/Hero.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from "next/image";
+import { HiOutlineCalculator } from "react-icons/hi2";
+
+
+export default function Hero() {
```

Comment:
there are better pattern for the exporting components.
Will be better to create index.js in this folder and do re-exporting of components from that file